mozilla_schema_generator/__main__.py

```diff
@@ -67,6 +74,79 @@ def generate_main_ping(config, out_dir, split, pretty):
     dump_schema(schemas, out_dir, pretty, version=4)
 
 
+@click.command()
+@click.argument(
+    'config-dir',
+    type=click.Path(
+        dir_okay=True,
+        file_okay=False,
+        writable=False,
+        exists=True,
+    ),
+    default=CONFIGS_DIR,
+)
+@click.option(
+    '--out-dir',
+    help=("The directory to write the schema files to. "
+          "If not provided, writes the schemas to stdout."),
+    type=click.Path(
+        dir_okay=True,
+        file_okay=False,
+        writable=True,
+    ),
+    required=False
+)
+@click.option(
+    '--split',
+    is_flag=True,
+    help=("If provided, splits the schema into "
+          "smaller sub-schemas"),
+)
+@click.option(
+    '--pretty',
+    is_flag=True,
+    help=("If specified, pretty-prints the JSON "
+          "schemas that are outputted. Otherwise "
+          "the schemas will be on one line."),
+)
+@click.option(
+    '--common-pings-config',
+    default="common_pings.json",
+    help=("File containing URLs to schemas and configs "
+          "of pings in the common ping format."),
+)
+def generate_common_pings(config_dir, out_dir, split, pretty, common_pings_config):
```

Comment:
I don't believe we need `generate_main_ping` any longer.

Reply:
For backwards compatibility and because main ping is handled slightly different from common pings, I added back `MainPing`. So to generate main ping we'd now still have to call `generate_main_ping`